Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/GraphSerialization.cpp

```diff
@@ -145,7 +145,11 @@ namespace ScriptCanvas
                 , AZ::ObjectStream::FilterDescriptor(nullptr, AZ::ObjectStream::FILTERFLAG_IGNORE_UNKNOWN_CLASSES)))
             {
                 result.m_isSuccessful = false;
-                result.m_errors = "JSON (and failsafe XML) deserialize attempt failed.";
+                result.m_errors = "JSON (and failsafe XML) deserialize attempt failed.\n";
+                result.m_errors += "The error might be caused by deprecated Spawnable Nodes in your target scriptcanvas file.\n";
+                result.m_errors += "If so, please run UpdateSpawnableNodes python script for the scriptcanvas file to see if the error is resolved.\n";
+                result.m_errors += "(Run 'python {Your o3de engine folder}\Gems\ScriptCanvas\SourceModificationScripts\UpdateSpawnableNodes.py {Your target scriptcanvas file}')";
+
```

Comment:
Unrelated to this PR, I wonder why we did not migrate the functionality of that script into a version converter and do it automatically for users.

Reply:
If I remember correctly, we want to update the SCs only when users want to.

Reply:
I don't think that's the reason. If users didn't upgrade the SC files, their spawnable nodes will be in an unusable state. So leaving the choice to the user is not a good idea. The reason a version convertor doesn't exist for this is because there is no default version convertor that the json serializer provides. So systems would need to write their own version convertors.